### PR TITLE
[GLUTEN-5335][VL] Use common name for both celeborn and uniffle

### PR DESCRIPTION
**Pull request by summaryzb**
## What changes were proposed in this pull request?
https://github.com/apache/incubator-gluten/issues/5335

## How was this patch tested?
integration test

### CODE REVIEW

**Comment by summaryzb** [comment]
@jackylee-ch @PHILO-HE PTAL

**Comment by jackylee-ch** [comment]
Maybe we should change the profile in pom.xml from `-rss -rss-uniffle` to `-celeborn -uniffle`.

**Comment by jackylee-ch** [comment]
also cc @Yohahaha

**Comment by Yohahaha** [comment]
cc @kerwin-zk

File: cpp/core/jni/JniCommon.h [hunk 1]
```diff
@@ -411,28 +411,28 @@ class BacktraceAllocationListener final : public gluten::AllocationListener {
   std::atomic_int64_t backtraceBytes_{1L << 30};
 };
 
-class CelebornClient : public RssClient {
+class JavaRssClient : public RssClient {
  public:
-  CelebornClient(JavaVM* vm, jobject javaCelebornShuffleWriter, jmethodID javaCelebornPushPartitionDataMethod)
-      : vm_(vm), javaCelebornPushPartitionData_(javaCelebornPushPartitionDataMethod) {
+  JavaRssClient(JavaVM* vm, jobject javaShuffleWriter, jmethodID javaPushPartitionDataMethod)
```
Comment (kerwin-zk) [inline cpp/core/jni/JniCommon.h:416]: javaShuffleWriter -> javaRssShuffleWriter

File: cpp/core/jni/JniCommon.h [hunk 2]
```diff
@@ -452,17 +452,16 @@ class CelebornClient : public RssClient {
...
+    return static_cast<int32_t>(javaBytesSize);
   }
 
   void stop() override {}
 
  private:
   JavaVM* vm_;
-  jobject javaCelebornShuffleWriter_;
-  jmethodID javaCelebornPushPartitionData_;
+  jobject javaShuffleWriter_;
```
Comment (kerwin-zk) [inline cpp/core/jni/JniCommon.h:464]: javaShuffleWriter_ -> javaRssShuffleWriter_

File: cpp/core/jni/JniCommon.h (hunk 1 above)
Comment (summaryzb) [reply cpp/core/jni/JniCommon.h]: OK

File: cpp/core/jni/JniCommon.h (hunk 2 above)
Comment (summaryzb) [reply cpp/core/jni/JniCommon.h]: OK

**Review by jackylee-ch — APPROVED** [review]
Thanks for your work. @summaryzb

**Review by PHILO-HE — COMMENTED** [review]
@kerwin-zk, please take a look again. If it looks good to you, please approve it. Thanks!

**Review by kerwin-zk — APPROVED** [review]
LGTM. Thanks!

**Comment by jackylee-ch** [comment]
Thanks for your review. @PHILO-HE @kerwin-zk @Yohahaha

**Pull request merged**